2011-01-12  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/et_EE: Change currency to euro.

---
 localedata/locales/et_EE |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/localedata/locales/et_EE
+++ b/localedata/locales/et_EE
@@ -109,8 +109,8 @@
 END LC_CTYPE
 
 LC_MONETARY
-int_curr_symbol     "<U0045><U0045><U004B><U0020>"
-currency_symbol     "<U006B><U0072>"
+int_curr_symbol     "<U0045><U0055><U0052><U0020>"
+currency_symbol     "<U20AC>"
 mon_decimal_point   "<U002C>"
 mon_thousands_sep   "<U00A0>"
 mon_grouping        3;3
